spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -287,6 +298,34 @@ public int getInstanceNumber() {
 			result.put("url", baseUrl.toString());
 			return result;
 		}
+
+		/**
+		 * Will start the process while redirecting 'out' and 'err' streams
+		 * to the 'out' and 'err' streams of this process.
+		 */
+		private void start(ProcessBuilder builder) throws IOException {
+			if (this.stdout == null) {
+				builder.redirectOutput(Redirect.INHERIT);
+				builder.redirectError(Redirect.INHERIT);
+			}
+			this.process = builder.start();
+		    this.pid = getLocalProcessPid(this.process);
+		}
+
+		private void start(ProcessBuilder builder, Path workDir, boolean deleteOnExist) throws IOException {
+			builder.directory(workDir.toFile());
```

Comment:
@ericbottard  I can go either way on this one, but i don't think it would hurt, so, let me polish some more and I'll make another push